apps/src/templates/instructions/ThreeColumns.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+/**
+ * A component that lays out children in three columns (left, center, right),
+ * where left/right widths are specified and center is the remainder. This is
+ * done largely to separate some of the semi-complicated styling.
+ */
+
+const ThreeColumns = (props) => {
+  const { style, leftColWidth, rightColWidth, height, children } = props;
+
+  const styles = {
+    container: {
+      paddingLeft: leftColWidth,
+      paddingRight: rightColWidth,
+      float: 'left',
+      width: '100%',
+      boxSizing: 'border-box'
+    },
+    middle: {
+      width: '100%',
+      position: 'relative',
+      float: 'left',
+      overflowY: 'scroll',
+      height
+    },
+    left: {
+      position: 'relative',
+      float: 'left',
+      width: leftColWidth,
+      right: leftColWidth,
+      marginLeft: '-100%',
+    },
+    right: {
+      position: 'relative',
+      float: 'left',
+      width: rightColWidth,
+      marginRight: -rightColWidth
+    }
+  };
+
+  return (
+    <div style={Object.assign({}, styles.container, style)}>
+      <div style={styles.middle}>{children[1]}</div>
+      <div style={styles.left}>{children[0]}</div>
+      <div style={styles.right}>{children[2]}</div>
+    </div>
+  );
+};
+
+export default ThreeColumns;
```

Comment:
Should maybe have a custom PropTypes validator that complains if there are not exactly three children?